_apps/delete-merged-branch.md

```diff
@@ -0,0 +1,38 @@
+---
+title: Delete merged branch
+description: No more manually deleting merged branches, this lovely app does it for you.
+slug: delete-merged-branch
+screenshots:
+- https://user-images.githubusercontent.com/24505883/42899819-7c05a47a-8ac7-11e8-8be9-9e3888f1bedc.gif
+authors:
+- svanboxel
+repository: svanboxel/delete-merged-branch
+host: https://delete-merged-branch.now.sh
+---
+
+# Probot: Delete merged branch
+
+> A GitHub app built with [Probot](https://github.com/probot/probot) that automatically deletes a branch after it's merged. That's it, enjoy! 
```

Comment:
Maybe we could rm this line?

We typically avoid repeating that apps are probot/github apps in the description, since it should be obvious since they're on the website :)